lib/ash_json_api/request.ex

```diff
@@ -226,7 +226,11 @@ defmodule AshJsonApi.Request do
 
             {:ok, "application", "vnd.api+json", params} ->
               Application.get_env(:ash_json_api, :allow_all_media_type_params?, false) ||
-                valid_header_params?(params)
+                valid_header_params?(params, :json)
+
+            {:ok, "multipart", "ash-json-api+form-data", params} ->
```

Comment:
This is interesting 🤔 I hesitate to "leak" the internal tool name here. I'd rather make something up that could *theoretically* be adopted manually. Something like `vnd.api+json-multipart`?

Reply:
I've changed it to `multipart/vnd.api+form-data`. I hesitated to use it since `vnd.api` is registered to jsonapi and this is not provided by them.

Reply:
🤔 Yeah, you're right. I thought they just had the whole thing but it's `vnd.<vendor>`.

What to do... My thinking was that it is that it's one thing to "leak" a given protocol that you're implementing to a client (you're already intentionally communicating that typically), and another thing to "leak" the actual underlying technology doing the implementation.

Wondering if there is a prefix *like* `vnd` that means "custom to the implementation".

Reply:
I think your original approach is better than leveraging a known reserved name. Lets go back to that, and if someone has a problem with it we can add a config to customize that content type.

Reply:
🧠 lets do it 👍

Reply:
Or going by [RFC 6838 §3.2](https://www.rfc-editor.org/rfc/rfc6838.html#section-3.2) we could also do this:

`multipart/vnd.ash+form-data`

Registration is encouraged but is not required.

Reply:
oh? Interesting 🤔 I think for now let's use the `x-ash`. My reasoning there is that we can potentially get this methodology established as an extension to the `json:api` spec itself, and/or establish it as its own light weight spec alternative or something, at which point we can register it and publish it separately.

Reply:
but now its a bit too likely to change in the future I think.

Reply:
@zachdaniel It seems RFC 2045 is not used anymore. Going by 6838 it should then be `multipart/x.ash+form-data`.

Reply:
😅 I'm very glad you are sleuthing this 🙇